pkg/features/kube_features.go

```diff
@@ -772,6 +774,14 @@ const (
 	//
 	// Enables Leader Migration for kube-controller-manager and cloud-controller-manager
 	ControllerManagerLeaderMigration featuregate.Feature = "ControllerManagerLeaderMigration"
+
+	// owner: @fromanirh
+	// alpha: v1.23
+	// beta: perma-alpha, see KEP for details
```

Comment:
Is this the terminoligy that was agreed upon for this? /cc @johnbelamaric

Reply:
The PR here reflects what was written in the KEP. We will finalize the terminology and how we want to handle the newly introduced experimental options after a follow-up discussion in the Production Readiness Subproject meeting on 22nd September. I have already added an agenda item for this [here](https://docs.google.com/document/d/10QkXwiZfBL7wBKHHFslooXvGDKHA75jkkgiIXuKFryM/edit#).

Reply:
Sounds good. Let's hold on this until we know what the final terminology will be then.

Reply:
We had the PRR subproject meeting, and no major objections were raised from PRR perspective; however, it was pointed out that this is effectively more a arch/api-review concern than a PRR concern, and to move the discussion on sig-arch proper. This is what me and @swatisehgal are currently doing. I will post updates as we have them.

Reply:
Email thread here: https://groups.google.com/u/1/g/kubernetes-sig-architecture/c/Nxsc7pfe5rw

Reply:
PR to align the KEP with the outcome of the discussion: https://github.com/kubernetes/enhancements/pull/2993